src/main/kotlin/org/rust/ide/intentions/JoinWildcardsIntention.kt

```diff
@@ -0,0 +1,61 @@
+/*
+ * Use of this source code is governed by the MIT license that can be
+ * found in the LICENSE file.
+ */
+
+package org.rust.ide.intentions
+
+import com.intellij.openapi.editor.Editor
+import com.intellij.openapi.project.Project
+import com.intellij.openapi.util.TextRange
+import com.intellij.psi.PsiElement
+import org.rust.lang.core.psi.*
+import org.rust.lang.core.psi.ext.ancestorStrict
+import org.rust.lang.core.psi.ext.endOffset
+import org.rust.lang.core.psi.ext.startOffset
+
+class JoinWildcardsIntention : RsElementBaseIntentionAction<List<RsPatWild>>() {
+    override fun getText(): String = "Replace `_` sequence with `..`"
```

Comment:
I also renamed the intention, But I don't like the new name too. Maybe "Replace repeating `_` with `..`"? But these names are not applicable for single `_`

Reply:
I think that `sequence` is fine, since sequence can also have length 1. But otherwise maybe `successive` instead of `repeating`?

Reply:
Ok, I made the name different for single or multiple `_` cases